features/input-file-multiple.yml.dist

```diff
@@ -0,0 +1,14 @@
+# Generated from: input-file-multiple.yml
+# Do not edit this file by hand. Edit the source file instead!
+
+status:
+  baseline: false
+  support:
+    chrome: "22"
+    chrome_android: "25"
+    edge: "79"
+    firefox: "50"
```

Comment:
Are you sure BCD is correct that this doesn't work in Firefox Android?

Reply:
Yes, that's correct.